src/Menu/index.js

```diff
@@ -150,11 +150,17 @@ Menu.propTypes = {
       name: PropTypes.string,
       value: PropTypes.string
     })
-  ).isRequired
+  ).isRequired,
+
+  /**
+   * Initial selected menu item value
+   */
+  selected: PropTypes.string
```

Comment:
@euanlau we probably want to remove the `state` `selected` property now that we're passing in selected as a prop and update `renderMenuItem` appropriately.

Reply:
Also, it mentions that `selected` is a string and the selected menu item. Does this represent the value of the menu item? The ID? We should probably update the doc here to be more specific. I imagine once the state is removed from the component, the test will also help identify what we mean here.

Reply:
That is modifying the behavior of Menu itself is out of the scope of this story I think?

Reply:
I just added the `PropTypes` since it was missing.